### PR TITLE
Support `expect.addSnapshotSerializer()`

### LINKED ISSUE #28: Feature request: Support `expect.addSnapshotSerializer()`
Not sure how it works.

### PR DESCRIPTION
**Pull request by fisker**
Fixes #28

~I haven't test~, but seems this is how it works. https://github.com/facebook/jest/blob/bb0956589f9dbddcc13db0f42b2d4d9ff642b2cd/packages/jest-expect/src/index.ts#L35

Tested, it works.

### CODE REVIEW

**Comment by SimenB** [comment]
This runner can probably import and use `@jest/expect` instead of `expect`, fwiw

**Comment by fisker** [comment]
Good idea, I'll have a test.

**Comment by SimenB** [comment]
`yarn dedupe` maybe? not sure what error you got

**Comment by fisker** [comment]
No error, but `expect.addSnapshotSerializer` added serializer didn't work. Added to somewhere else? 

> yarn dedupe

Can't, haven't updated Jest.

**Review by nicolo-ribaudo — APPROVED** [review]
`@jest/expect` is just a tiny wrapper around `expect` (https://unpkg.com/browse/@jest/expect@28.0.2/build/index.js), so it shouldn't cause any problem that we cannot already have with the direct dependency on `expect`

**Comment by nicolo-ribaudo** [comment]
I'm assuming the tests will pass on `main` :smile:

File: src/global-setup.js [hunk 1]
```diff
@@ -1,18 +1,10 @@
 /* eslint-disable import/extensions */
 
 import mock from "jest-mock";
-import { expect } from "expect";
+import { jestExpect as expect } from "@jest/expect";
 import snapshot from "jest-snapshot";
```
Comment (SimenB) [inline src/global-setup.js:5]: unused

Comment (nicolo-ribaudo) [reply src/global-setup.js:5]: Thanks! Fixed in 9b0a402ca1832ac14c3aa4453969fbeabb6dd187

**Pull request merged**